<testsuites duration="4.271">
<testsuite name="integration" tests="1" failures="1" time="4.271" id="0">
<testcase name="zero-arg-function" time="4.268" classname="zero-arg-function">
<failure type="test output mismatch" message="test output mismatch">
0a1,13
&gt; uncaught exception System.Exception
&gt; 	execvp: binary: Unknown error 24832016
&gt; ../../../tester/tester(_ZN6System9Exception13initBacktraceEv+0x3a) [0x4a485a]
&gt; ../../../tester/tester(_ZN6System9Exception4initEN6System6StringE+0x22) [0x4a47a2]
&gt; ../../../tester/tester(_ZN4Util7Process14throwUnixErrorEN6System6StringE+0x67) [0x4a0117]
&gt; ../../../tester/tester(_ZN4Util7Process5spawnEPcPPcPPc+0x5e) [0x4a040e]
&gt; ../../../tester/tester(_ZN4Util7Process5spawnEN6System6StringEA_N6System6StringEA_N6System6StringE+0x1c1) [0x4a05e1]
&gt; ../../../tester/tester(_ZN6Tester11TEST_RUNNER4testEv+0x908) [0x47f7a8]
&gt; ../../../tester/tester(_ZN6Tester4Main4initEv+0x417) [0x491627]
&gt; ../../../tester/tester(_ZN6System7Startup4run1EN6System6ObjectE+0x1f) [0x48a47f]
&gt; ../../../tester/tester(__entry_point+0x20) [0x498a80]
&gt; ../../../tester/tester(main+0x29) [0x46df59]
&gt; 
</failure>
<system-out>uncaught exception System.Exception
	execvp: binary: Unknown error 24832016
../../../tester/tester(_ZN6System9Exception13initBacktraceEv+0x3a) [0x4a485a]
../../../tester/tester(_ZN6System9Exception4initEN6System6StringE+0x22) [0x4a47a2]
../../../tester/tester(_ZN4Util7Process14throwUnixErrorEN6System6StringE+0x67) [0x4a0117]
../../../tester/tester(_ZN4Util7Process5spawnEPcPPcPPc+0x5e) [0x4a040e]
../../../tester/tester(_ZN4Util7Process5spawnEN6System6StringEA_N6System6StringEA_N6System6StringE+0x1c1) [0x4a05e1]
../../../tester/tester(_ZN6Tester11TEST_RUNNER4testEv+0x908) [0x47f7a8]
../../../tester/tester(_ZN6Tester4Main4initEv+0x417) [0x491627]
../../../tester/tester(_ZN6System7Startup4run1EN6System6ObjectE+0x1f) [0x48a47f]
../../../tester/tester(__entry_point+0x20) [0x498a80]
../../../tester/tester(main+0x29) [0x46df59]

</system-out>
<system-err>IO.IOException: open &quot;../lib/ghul.ghul&quot;: No such file or directory (2)
ghul(_ZN2IO11IOException4initEiN6System6StringE+0x2d) [0x7bb95d]
ghul(_ZN2IO10FileStream16throwIOExceptionEN6System6StringEiN6System6StringE+0x105) [0x7b8f55]
ghul(_ZN2IO4File10openStreamEN6System6StringEiib+0x62) [0x81fed2]
ghul(_ZN6Driver4Main4initEv+0x7a5) [0x79d765]
ghul(_ZN6System7Startup4run1EN6System6ObjectE+0xd) [0x827bad]
ghul(main+0x43) [0x739153]

</system-err>
</testcase>
</testsuite>
</testsuites>

